tests/utils.py

```diff
@@ -0,0 +1,20 @@
+# SPDX-License-Identifier: BSD-2-Clause-Patent
+# SPDX-FileCopyrightText: 2020 the prplMesh contributors (see AUTHORS.md)
+# This code is subject to the terms of the BSD+Patent license.
+# See LICENSE file for more details.
+
+from typing import Union
+
+import environment as env
+
+
+class Utils(object):
+    """Collection of common functions used by tests."""
+
+    @staticmethod
+    def check_log(entity_or_radio: Union[env.ALEntity, env.Radio], regex: str,
```

Comment:
Personally, I much prefer the approach taken in current test_flows.py: define a base class that has these helper functions, and make the individual tests derive from that class. But I guess it's mostly a matter of taste. It does become important, however, if we ever need to keep any state - and I think we *will* need to do that.